lib/compat/wordpress-6.7/blocks.php

```diff
@@ -103,3 +103,31 @@ static function ( $format ) {
 	return $query;
 }
 add_filter( 'query_loop_block_query_vars', 'gutenberg_add_format_query_vars_to_query_loop_block', 10, 2 );
+
+/**
+ * Adds `ignore` option for sticky posts to the Query block.
+ *
+ * @see 'query_loop_block_query_vars'
+ *
+ * @param array    $query The query vars.
+ * @param WP_Block $block Block instance.
+ * @return array   The filtered query vars.
+ */
+function gutenberg_add_ignore_sticky_posts_to_query_loop_block( $query, $block ) {
```

Comment:
As @carolinan mentioned, this should be under `wordpress-6.8` folder (you'll need to rebase). Just create a `blocks.php` file there with this function and include the file in `lib/load.php`.

Reply:
@ntsekouras okay done

Reply:
Thanks! You can remove these changes from this file now.

Reply:
@ntsekouras done!